### ISSUE TITLE
Feature Request: Add an option to setup a situation and ask which Icons are equal 

### ISSUE THREAD

**Issue by Mannshoch** [westnordost/StreetComplete#264]
Add an option to turn around the Question system. 
Ask at first for the surface and the Material and ask after that which of the balloon are Equal to it.

It's otherwise a pain to setup street surface for a whole village.

**Comment by matkoniecz** [comment]
For start - you should not add data based on guess, you should visit each place (in my city I though about mass setting all roads to asphalt, during surveys I discovered that it would introduce some untrue data like at https://www.openstreetmap.org/?mlat=50.07091&mlon=19.92517#map=19/50.07091/19.92517)

**Comment by matkoniecz** [comment]
Also, it is a duplicate of https://github.com/westnordost/StreetComplete/issues/73

**Comment by Mannshoch** [comment]
Nothing to do with Gues. The Big mainstreet througth my hometown is asphalt but it seems splitted into little pieces. Also the City Trip I did. Most of the town is asphalt but this Information seems to be missing on every road. And also there it seems all Streets are split into little pieces. I looked over a litle crossing next to an expressway. I had to select 12 times a bubble to change it to asphalt. the whole Expressway between Two Citys I looked over was also not setup and I do not why but is also splitted into some Pieces. 
Such kind of Work kills the motivation to do something.
